test/jest.config.js

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+process.env.TZ = 'UTC';
+
+module.exports = {
```

Comment:
does this have something that we dont have available already within the global jest config?

https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/dev/jest/config.js

Reply:
i don't think global config applies to plugin tests?